daprdocs/content/en/operations/configuration/environment-variables-secrets.md

```diff
@@ -0,0 +1,114 @@
+---
+type: docs
+title: "How-To: Configure Environment Variables from Secrets for Dapr sidecar"
+linkTitle: "Environment Variables from Secrets"
+weight: 7500
+description: "Inject Environment Variables from Kubernetes Secrets into Dapr sidecar"
+---
+In special cases, Dapr sidecar needs an environment variable injected into it. This use case may be required by a Component, a 3rd party library, or a module that uses environment variables to configure the said Component or customize its behavior. This can be useful for both production and non-production environments.
```

Comment:
```suggestion
In special cases, the Dapr sidecar needs an environment variable injected into it. This use case may be required by a component, a 3rd party library, or a module that uses environment variables to configure the said component or customize its behavior. This can be useful for both production and non-production environments.
```